modules/analysis-common/src/main/java/org/elasticsearch/analysis/common/ElisionTokenFilterFactory.java

```diff
@@ -35,6 +35,9 @@
 
     ElisionTokenFilterFactory(IndexSettings indexSettings, Environment env, String name, Settings settings) {
         super(indexSettings, name, settings);
+        if (settings.get("articles") == null) {
```

Comment:
It's hidden but it's also possible to set `articles_path` to reference a file so we should also check this alternative.
I am not sure that a lot of users are aware of this and I doubt that it is very useful but we should handle it for bwc.

Reply:
`articles_path` isn't documented, I'll add something to the tokenfilter docs about it as well.